pkg/service/repair/repair_test.go

```diff
@@ -124,164 +125,185 @@ func TestShouldRepairRing(t *testing.T) {
 		"h9": "dc4",
 	}
 
+	rs := func(reps ...string) scyllaclient.ReplicaTokenRanges {
+		return scyllaclient.ReplicaTokenRanges{
+			ReplicaSet: reps,
+			Ranges: []scyllaclient.TokenRange{
+				{
+					StartToken: dht.Murmur3MinToken,
+					EndToken:   dht.Murmur3MaxToken,
+				},
+			},
+		}
+	}
+
+	allDCs := []string{"dc1", "dc2", "dc3", "dc4"}
+
 	testCases := []struct {
+		Name     string
 		Ring     scyllaclient.Ring
 		DCs      []string
 		Host     string
 		Expected bool
 	}{
 		{
+			Name: "LocalStrategy",
 			Ring: scyllaclient.Ring{
-				HostDC:      hostDC,
-				Replication: scyllaclient.LocalStrategy,
-				RF:          1,
+				ReplicaTokens: []scyllaclient.ReplicaTokenRanges{rs("h1")},
+				HostDC:        hostDC,
 			},
-			DCs:      []string{"dc1", "dc2", "dc3", "dc4"},
-			Expected: false,
+			DCs:      allDCs,
+			Expected: false, // can't repair local keyspace
 		},
 		{
+			Name: "SimpleStrategy{rf=1}",
 			Ring: scyllaclient.Ring{
-				HostDC:      hostDC,
-				Replication: scyllaclient.SimpleStrategy,
-				RF:          1,
+				ReplicaTokens: []scyllaclient.ReplicaTokenRanges{
+					rs("h1"), rs("h2"), rs("h3"),
+					rs("h4"), rs("h5"), rs("h6"),
+					rs("h7"), rs("h8"), rs("h9"),
+				},
+				HostDC: hostDC,
 			},
-			DCs:      []string{"dc1", "dc2", "dc3", "dc4"},
-			Expected: false,
+			DCs:      allDCs,
+			Expected: false, // can't repair rf=1
 		},
 		{
+			Name: "SimpleStrategy{rf=2}",
```

Comment:
But this is not true. The RF per DC is not 2 here.
Besides rf=2 is a very much shortcut here, as the replication factor is set per datacenter.
Please either remove this test-case or provide realistic replica sets for this test.